### PR TITLE
Removed duplicated definitions

### PR DESCRIPTION
**Pull request by Piterden**

### CODE REVIEW

File: core/replicators.js [hunk 1]
```diff
@@ -1,21 +1,10 @@
...
 
 module.exports = {
-  copyMethods: {
-    audio: 'sendAudio',
-    contact: 'sendContact',
-    document: 'sendDocument',
-    location: 'sendLocation',
-    photo: 'sendPhoto',
-    sticker: 'sendSticker',
-    text: 'sendMessage',
```
Comment (wojpawlik) [inline core/replicators.js:11]: Forgot `"text" -/> "sendText"`

Comment (Piterden) [reply core/replicators.js:11]: Oh! Moment...

File: core/replicators.js [hunk 2]
```diff
@@ -1,21 +1,10 @@
...
-    video: 'sendVideo',
-    video_note: 'sendVideoNote',
-    animation: 'sendAnimation',
-    voice: 'sendVoice',
-    poll: 'sendPoll'
-  },
+  send: (method) => `send${method
+    .split('_')
+    .map((word) => `${word.charAt(0).toUpperCase()}${word.slice(1)}`)
+    .join('')}`,
```
Comment (wojpawlik) [inline core/replicators.js:7]:
```js
function pascalCase(s) {
    return s.replace(/_(\w)/g, ([, L]) => L.toUpperCase())
}
```

Comment (Piterden) [reply core/replicators.js:7]: image

Comment (Piterden) [reply core/replicators.js:7]: image

Comment (Piterden) [reply core/replicators.js:7]: image

Comment (wojpawlik) [reply core/replicators.js]:
```js
pascalCase(`send_${method === 'text' ? 'message' : method}`)
```

**Comment by Piterden** [comment]
@wojpawlik thanx for help)

**Comment by wojpawlik** [comment]
Np :) I'd still prefer to have explicit `camelCase` or `pascalCase` for readability.

File: core/replicators.js (hunk 2 above)
Comment (wojpawlik) [reply core/replicators.js]: I actually suggested `camelCase`, my bad

**Comment by Piterden** [comment]
@wojpawlik me too, but I don't know where to place it(

**Comment by wojpawlik** [comment]
Just, above the export 🤷‍♂️

File: core/replicators.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
 const { formatHTML } = require('../markup')
 
+const pascalCase = (string) => string.split('_')
+  .map(([first, ...other]) => `${first.toUpperCase()}${other.join('')}`)
```
[the remark this answers is not in the document]
Comment (Piterden) [reply core/replicators.js:4]: https://jsperf.com/pascalcase-competition/1

**Pull request closed without merging**